### PR TITLE
Check terminologies

### PR DESCRIPTION
**Pull request by mrinnetmaki**
I believe the coding should state both the system and the code. @Marvasuo and @mikajylha please check the proposed fixes.

### CODE REVIEW

File: input/examples/organization-Toimintayksikko [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "name": "Esko-Testi pp HVA",
     "telecom": [
         {
-            "system": "phone",
+            "system": "http://hl7.org/fhir/contact-point-system",
+            "code": "phone",
             "value": "0404040404"
         }
```
Comment (Marvasuo) [inline input/examples/organization-Toimintayksikko:28]: Telecom is data type of ContactPoint and it doesn't contain "code" value.
https://www.hl7.org/fhir/datatypes.html#ContactPoint

File: input/fsh/examples/patientOfMunicipality.fsh [hunk 2]
```diff
@@ -12,8 +12,10 @@ Usage: #example
 * name
   * given = "Testi"
   * family = "Henkilö"
-* telecom.system = #phone
-* telecom.value = "+358401234657"
+* telecom
+  * system = "http://hl7.org/fhir/contact-point-system"
+  * code = #phone
+  * value = "+358401234657"
```
Comment (Marvasuo) [inline input/fsh/examples/patientOfMunicipality.fsh:18]: Same issue as above.

Comment (mrinnetmaki) [reply input/fsh/examples/patientOfMunicipality.fsh]: Agreed and fixed accordingly.

File: input/examples/organization-Toimintayksikko (hunk 1 above)
Comment (mrinnetmaki) [reply input/examples/organization-Toimintayksikko]: Agreed and fixed accordingly.

**Review by Marvasuo — APPROVED** [review]

**Pull request merged**